topics/introduction/tutorials/galaxy-intro-101/metadata.yaml

```diff
@@ -16,7 +16,7 @@ objectives:
 - "Learn how to share your work"
 time_estimation: "1-1.5h"
 key_points:
-- "Galaxy provides an easy-to-use graphical user interface for often complex commandline tools"
+- "Galaxy provides an easy-to-use graphical user interface for often complex command-line tools"
```

Comment:
Can you rather modify the main `metadata.yaml` file in `introduction` directory? It is the one that is used...

Reply:
Sorry, this is a bit confusing. Should the redundant metadata be removed from this file? If yes, what should be left here?

Reply:
This file is supposed to be removed (PR in progress 😉). Sorry for the confusion